weak03/homework03.py

```diff
@@ -0,0 +1,42 @@
+import requests
+from bs4 import BeautifulSoup
+
+print("날짜를 입력하세요")
+date = input()
+
+headers = {
+    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64)AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36'
+}
+
+response = requests.get(
+    'https://www.genie.co.kr/chart/top200?ditc=D&rtm=N&ymd=' + date
+)
```

Comment:
```suggestion
response = requests.get(
    'https://www.genie.co.kr/chart/top200?ditc=D&rtm=N&ymd=' + date,
    headers=headers
)
```
위와 같이 user-agent 헤더를 추가해주면 응답을 받을 수 있을 것 같습니다.